ocs_ci/utility/utils.py

```diff
@@ -5197,3 +5198,38 @@ def extract_image_urls(string_data):
     # Find all URLs that start with 'registry.redhat.io'
     image_urls = re.findall(r'registry\.redhat\.io[^\s"]+', string_data)
     return image_urls
+
+
+def params_to_configini_file(params):
+    """
+    This function will create a config ini file for the params given.
```

Comment:
Add a space above this line, this may cause a tox -e docs error

Reply:
But all the other functions have the docs written in the same pattern

Reply:
We always have the empty line before the (Args, Returns,  Raises) kind of keywords, I did below  change and reran the `tox -e docs` and it pass successfully. before that it was failing
```
    This function will create a config ini file for the params given.
    
    Use case:
        can be used in external-cluster-exporter script --config-file argument

```

```
❯ tox -e docs
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/html.
___________________________________________________________________ summary ____________________________________________________________________
  docs: commands succeeded
  congratulations :)
 ```

Reply:
I tried this but black prechecks is removing the extra line 
(venv-ocsci-311) [sdurgbun@corpha2-int ocs-ci (test/RHSTOR-5260 ✗)]$ git commit --signoff -m "Tox fix"
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted ocs_ci/utility/utils.py

All done! ✨ 🍰 ✨
1 file reformatted.

Check JSON...........................................(no files to check)Skipped
Check Yaml...........................................(no files to check)Skipped
Debug Statements (Python)................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
flake8...................................................................Passed
(venv-ocsci-311) [sdurgbun@corpha2-int ocs-ci (test/RHSTOR-5260 ✗)]$ git diff
(venv-ocsci-311) [sdurgbun@corpha2-int ocs-ci (test/RHSTOR-5260 ✗)]$
diff --git a/ocs_ci/utility/utils.py b/ocs_ci/utility/utils.py
index 9ec68ba80..8dbba66ef 100644
--- a/ocs_ci/utility/utils.py
+++ b/ocs_ci/utility/utils.py
@@ -5202,7 +5202,7 @@ def extract_image_urls(string_data):

 def params_to_configini_file(params):
     """
-
+
     This function will create a config ini file for the params given.
     Use case:
         can be used in external-cluster-exporter script --config-file argument
(END)